subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -67,6 +67,9 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 /* Leave room for 32 hexadeciaml digits (UUID) + NULL */
 #define CLIENT_EP_LEN		33
 
+/* Up to 3 characters + NULL */
+#define CLIENT_BINDING_LEN 4
```

Comment:
If we have such a constraint, then this could be also written as `#define CLIENT_BINDING_LEN sizeof("abc");`
Or we could automatically use the longest string here like this `#define CLIENT_BINDING_LEN sizeof("UQ");`

Reply:
Fixed, I've used the longest possible binding mode  string.